### ISSUE TITLE
I followed the guide to use the server but when I click "Play" I still go to the official Slither.io server

### ISSUE THREAD

**Issue by crazyyi** [circa94/JSlither#15]
Am I missing some step?

I put this line :`sos = [{ip: "127.0.0.1", po: 8080, ac: 132, ptm: 12}];`
into my browser console already. And I have updated the config.json to use ip 127.0.0.1.

**Comment by Kogs** [comment]
Maybe something in the game changed. But make sure you did not pressed play bevor changing the `sos`. Try to refresh your browser page then insert the sos then press play.

**Comment by crazyyi** [comment]
@Kogs I have tried several times but no luck.

**Comment by Kogs** [comment]
Yes the game has changed something. And its not working anymore. If I find the time i will take a look at it. But for now you can just google for any slither plugin that allows you to join a server ip directly.
